src/nicypkg/functions.nim

```diff
@@ -81,10 +81,10 @@ proc horizontalRule*(c: char = '-'): string =
   result &= zeroWidth("\n")
 
 proc tilde*(path: string): string =
-  # borrowed from @jabbalaci
-  let home = getHomeDir().rchop("/")    # without trailing '/'
+  # donated by @misterbianco
+  let home = getHomeDir()
   if path.startsWith(home):
-    result = "~" & path.lchop(home)    # replace the first occurrence only
+    result = "~/" & path.split(home)[1]
```

Comment:
As far as I can tell, this will break paths that have `/home/user/foo/home/user/bar`, resulting in `~/foo/~/bar`. Look at #1.

Reply:
Oh wait. I think I'm wrong.

Reply:
Yeah I think this looks good. Add a commit that deletes `src/nicypkg/utils.nim`, will you? That's redundant code.
We can merge this after.